src/main/java/org/moltimate/moltimatebackend/service/LigandService.java

```diff
@@ -121,15 +161,33 @@ public static MultipartFile fetchLigand(String ligandID) {
      * @throws PDBFetchException
      */
     @Retryable(maxAttempts = 2, backoff = @Backoff(5000))
-    public String getEcNumber(Structure structure) throws InvalidPdbIdException {
-        RCSBDescription description = RCSBDescriptionFactory.get(structure.getPDBCode());
-        if (description == null) {
-            throw new InvalidPdbIdException(structure.getPDBCode());
+    public String getEcNumber(Structure structure) throws InvalidPdbIdException, IOException {
+        String USER_AGENT = "Mozilla/5.0";
```

Comment:
This seems like something that should exist globally

Reply:
Agreed, that will be part of the refactoring of the architecture decisions Mark had come to during this sprint.